apps/apputil.hpp

```diff
@@ -76,7 +76,7 @@ inline void SysCleanupNetwork() {}
 // See:
 //    https://msdn.microsoft.com/en-us/library/windows/desktop/ms742214(v=vs.85).aspx
 //    http://www.winsocketdotnetworkprogramming.com/winsock2programming/winsock2advancedInternet3b.html
-#ifdef __MINGW32__
+#ifndef HAVE_INET_PTON
 static inline int inet_pton(int af, const char * src, void * dst)
```

Comment:
This particular inet_pton() implementation is Windows-only. Shouldn't there be `#ifdef _WIN32` around it too, maybe just below `#ifndef HAVE_INET_PTON`?

Reply:
Sure.